specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json

```diff
@@ -2550,14 +2750,56 @@
     },
     "ApplicationCreateParameters": {
       "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Application"
```

Comment:
No, let's discuss this, we spent so much time to refactor this hiearchy with Amar, the allOf is not a good idea

Reply:
Like I mentioned in an early comment, I have made the change, please take another look.